src/style.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
```

Comment:
In here set up eaching heading's style - later on if you want to make all h1's into text-10xl you'll only edit one single line instead of hundreads
```
  h1 {
    @apply text-6xl text-black font-extrabold;
  },
  ...
```

Reply:
done